remove-webhook/remove-webhook.sh

```diff
@@ -1,24 +1,81 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
-if [ -n "$DEBUG" ]
-then
-    set -x
+usage() {
+  cat << EOF
+usage: bash remove-webhook.sh [--insecure-skip-tls-verify]
+
+This script adjusts the rancher-webhook chart release in all clusters managed by Rancher (excluding the local cluster).
+Depending on the version of Rancher, it either deletes the downstream webhook release, adjusts its version and restarts, or does nothing.
+Requires kubectl and helm to be installed and available on \$PATH.
+
+RANCHER_URL without a trailing slash must be set with the server URL of Rancher.
+RANCHER_TOKEN must be set with an admin token generated with no scope.
+To ignore TLS verification, set --insecure-skip-tls-verify.
+
+Users also need to ensure they have the rancher-charts repo in the local Helm index.
+EOF
+}
+
+if [ "$1" == "-h" ]; then
+  usage
+  exit 0
 fi
 
-usage() {
-    echo "./remove-webhook.sh [--insecure-skip-tls-verify]"
-    echo "Remove the webhook chart in all clusters managed by rancher (excluding the local cluster)"
-    echo "Requires kubectl and helm to be installed and available on \$PATH"
-    echo "--insecure-skip-tls-verify can be set to configure the script to ignore tls verification"
-    echo "RANCHER_TOKEN must be set with an admin token generated with no scope"
-    echo "RANCHER_URL must be set with the url of rancher (no trailing /) - should be the server URL"
+delete_webhook() {
+  cluster="$1"
+  current_chart=$(helm list -n cattle-system -l name=rancher-webhook | tail -1 | cut -f 6)
+  echo "Deleting $current_chart from cluster $cluster."
```

Comment:
Nit: given that we don't use the output from current chart at all, it might work just fine to log that we are deleting the rancher-webhook release and delete it, saving the extra list call.

Reply:
Yes, I am not a fan of parsing the last column of the output with the chart version, this looks brittle. But I don't know how else to get the current webhook version to display for the users. Or is including the version being deleted in the log unnecessary?

Reply:
It's probably good to include the version being uninstalled for audit purposes, so we can leave this as is. We can consider this read resolved.